jsonnet/components/node-exporter.libsonnet

```diff
@@ -350,4 +389,16 @@ function(params)
         ],
       },
     },
+
+    acceleratorsCollectorConfigmap: {
+      apiVersion: 'v1',
+      kind: 'ConfigMap',
+      metadata: {
+        name: acceleratorsConfigMapName,
+        namespace: cfg.namespace,
+      },
+      data: {
+        'accelerators-collector-config.yaml': std.manifestYamlDoc(acceleratorsConfigData),
```

Comment:
we could use `acceleratorsConfigFileName` here I think.

Reply:
in case it is a key, jsonnet does not substitute the variable for its value, but uses the name of the variable directly as key name

Reply:
fixed